lib/main.js

```diff
@@ -16,6 +16,8 @@
 
 'use strict'
 
+global.Promise = require('bluebird')
```

Comment:
Can we require `bluebird` in every necessary file instead? I'm not particularly a fan of declaring global variables.

Reply:
I can see why you would not be very fond of this pattern.